README.md

```diff
@@ -223,7 +223,7 @@ Julia does not install anything outside the directory it was cloned into. Julia
 
 ### ARM (Linux)
 
-See [README.arm](https://github.com/JuliaLang/julia/blob/master/doc/README.arm.md) for building ARM processors.
+See [README.arm](https://github.com/JuliaLang/julia/blob/master/doc/README.arm.md) for building on ARM processors.
```

Comment:
And here I was all this time thinking that README.arm would tell me how to manufacture ARM processors.

Reply:
good idea, we should include instruction on manufacturing arm processors as well 😛